tst/testinstall/opers/LocationFunc.tst

```diff
@@ -6,8 +6,8 @@ gap> LocationFunc(f);
 "stream:1"
 
 # Library function
-gap> LocationFunc(Where);
-"GAPROOT/lib/error.g:79"
+gap> PositionSublist(LocationFunc(Where),"/lib/error.g:") <> fail;
```

Comment:
Wait, that doesn't look like you are actually reverting the commit; you are partially reverting it, but also keep parts in modified form. IMHO the commit message should then not say "Revert"; or perhaps there should be two commits: one pure revert commit (with the message augmented by an explanation as to why the revert happens), and then a second commit which adds this test back, in its modified form.

Reply:
Alternative, just revert, and drop this test: not sure how valuable it is to begin with?